frontend/src/components/details/navbar.css

```diff
@@ -48,38 +48,39 @@
 /* Navigation links */
 .navbar addbtn {
     float: left;
-    padding: 12px;
+    padding: 15px;
     /* color: var(--global--panel--color); */
     background-color: var(--global--button--color);
     color: var(--global--button--text--color);
     text-decoration: none;
     font-size: 22px;
-    width: 20%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
-    text-align: right; /* If you want the text to be centered */
+    width: 5%; /* Four equal-width links. If you have two links, use 50%, and 33.33% for three links, etc.. */
```

Comment:
I see ... this is how the proportions of the nav bar are defined.

Reply:
and maybe we can change that somewhat. In a way where not the whole whitespace is linking, and the buttons react as expected when resizing the page. https://github.com/amosproj/amos-ss2021-carbon-footprint/pull/123#discussion_r654427843

Reply:
we have already defined  the navbar in the back , I guess it is possible to resize the button sizes but lets see how that works at the end 😄